README.md

```diff
@@ -1,5 +1,9 @@
 ## Getting Started
 
+You will need to set up your local environment/device using `eas device:create` in order to enable hot reloading. This is because the audio library we are using requires some native modules that must be built separately from the React app itself. If developing on iOS, this setup process might also require you have an Apple Developer account with the device(s) you are developing on registered to that account. More details [here](https://docs.expo.dev/eas-update/expo-dev-client/).
```

Comment:
This is needed to do any local development and not to just enable hot reloading. I don't think you can install development builds without doing this.

Reply:
Changed wording